pages/troubleshooting/error.md

```diff
@@ -15,7 +15,7 @@ If the Raspberry pi is unable to connect to the resin.io servers, the `ACT` LED
 
 This is either because it is not connected to the network or because the network ports which resin.io relies on are blocked in some way.
 
-* The first things to check in this case is that your device is correctly connected to ethernet or that you correctly entered the wifi credentials. To check that the wifi credentials are correct, you can check the `config.json` file on the `resin-boot` partition of the SD card (since Resin OS 1.2, or `resin-conf` for earlier versions).
+* The first things to check in this case is that your device is correctly connected to ethernet or that you correctly entered the wifi credentials. To check wifi credentials, power your device down, remove the SD card, and mount the SD card on your personal computer. If your device is running resinOS version 2.0 or greater, wifi credentials are listed in `system-connections/resin-wifi`, found in the `resin-boot` partition of the SD card. Otherwise, check the `config.json` file (in the `resin-boot` partition for versions 1.2 and greater, or `resin-conf` for earlier versions).
```

Comment:
One note here is that any device that is a `flasher-type` its not possible to inspect the running device boot partition (because its inside the device), however they can probably check the SD card anyways, since that would have the "initial configuration"